SolidStateGroup/bullet-train-api
# chore: Make features, identities and audit search, closed/open change requests filter based on URL

Thanks for submitting a PR! Please check the boxes below:


## Changes

In response to the audit UX and possibility of the features modal being a page, the following pages will now filter based on url params
- Features page
- Identity page
- Audit Log Page
- Change Request Page

As a result, it made sense to remove local storage from filter values, these concepts would only conflict and cause confusion.


## How to test

Apply every possible filter on each of the pages, check that they persist on refresh.
